stablehlo/integrations/python/tests/stablehlo.py

```diff
@@ -20,7 +20,6 @@
 from mlir import ir
 from mlir.dialects import stablehlo
 
-
```

Comment:
Typo: need two lines separating imports from functions and functions from each other.